feature/experimental/p4rt/ate_tests/google_discovery_protocol_packetin_test/google_discovery_protocol_packetin_test.go

```diff
@@ -0,0 +1,281 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package google_discovery_protocol_packetin_test
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"sort"
+	"testing"
+
+	"github.com/cisco-open/go-p4/p4rt_client"
+	"github.com/cisco-open/go-p4/utils"
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/telemetry"
+	"github.com/openconfig/ygot/ygot"
+	p4_v1 "github.com/p4lang/p4runtime/go/p4/v1"
+)
+
+const (
+	ipv4PrefixLen = 30
+	deviceId      = uint64(1)
+	portId        = uint32(10)
+	electionId    = uint64(100)
+)
+
+var (
+	p4InfoFile            = flag.String("p4info_file_location", "../../wbb.p4info.pb.txt", "Path to the p4info file.")
+	p4rtNodeName          = flag.String("p4rt_node_name", "0/1/CPU0-NPU1", "component name for P4RT Node")
+	streamName            = "p4rt"
+	gdpMAC                = "00:0a:da:f0:f0:f0"
```

Comment:
Are the gdpSrc MAC and gdpDst Mac have been hardcoded expected to always be these values as they are hardcoded?

Reply:
I was told gdpDst Mac and gdpEtherType are fixed value. @xw-g could you please help confirm if GDP destination MAC and EtherType are fixed values?